docs/UserGuide.md

```diff
@@ -404,11 +404,91 @@ Examples:
 * `importcsv n/2`
   * Reads from CSV treating the 2nd row as names, 3rd row as phone numbers, 4th row as email address, 5th row as tagged.
 
-
 ### Archiving data files `[coming in v2.0]`
 
 _Details coming soon ..._
 
+###Extension on Find contact: `find` `t/`
```

Comment:
You should combine this with the find command above in the user guide.
Also note the spacing of the colon (should match the other headers in this file).

Reply:
Note trailing ### too